resources/benchmark-runner.mjs

```diff
@@ -116,6 +116,106 @@ class PageElement {
     }
 }
 
+class MeasureTask {
+    constructor(runner, suite, test, page, frame) {
+        this._runner = runner;
+        this.suite = suite;
+        this.test = test;
+        this._page = page;
+        this._frame = frame;
+
+        this._unusedHeight = undefined;
+
+        this._wasRun = false;
+
+        this._asyncMeasurePromise = new Promise((resolve) => {
+            this._asyncDoneCallback = resolve;
+        });
+        this._measureSyncCallback = this._measureSync.bind(this);
+        this._measureAsyncTimeoutCallback = this._measureAsyncTimeout.bind(this);
```

Comment:
Since we're no longer using setTimeout, maybe we should call this._measureAsyncCallback?